concordium-sdk/src/main/java/com/concordium/sdk/crypto/wallet/WordLists.java

```diff
@@ -0,0 +1,2058 @@
+package com.concordium.sdk.crypto.wallet;
+
+public class WordLists {
+    /*
+     * The English BIP39 wordlist as taken from
+     * https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt.
+     * This wordlist is in code as it is supplied as a resource file in the BIP39
```

Comment:
It could be added as a resource but by having the list in code means we don't have to do any special Android handling. At this time I think I slightly prefer just having it in-code, but if you think it is nicer to add as a resource for Android, then I can take a look at that.